public/css/style.css

```diff
@@ -1,117 +1,35 @@
 body {
   background-image: url('../background.png');
```

Comment:
Do you like the background? I think it could be improved.

Reply:
I don't mind it. It's not great but it doesn't distract me too much. And other options option will be plain color background.